README.md

```diff
@@ -1 +1,60 @@
 # dm-borrower-frontend
+
+The Borrower Frontend provides a website for Borrower to view and sign their
+mortgage deeds online.
+
+## Contents
+- [Usage](#usage)
+- [Getting Started](#getting-started)
+- [Acceptance Tests](#acceptance-tests)
+
+## Usage
+```
+GET     /health                         -- renders standard Gov UK template
+GET     /searchdeed                     -- borrower views deed landing page
+POST    /searchdeed/search              -- form POST the response is a html page which will display the deed or an error page
+```
+
+Install the requirements
+```
+pip install -r requirements.txt
+pip install -r requirements_test.txt
+```
+
+Optional: export variable for deed-api
+```
+export DEED_API_BASE_HOST=http://localhost:8000
+```
+
+> default is localhost:9030
+
+Run the unit tests
+```
+source tests.sh
```

Comment:
I think you're looking for this https://github.com/LandRegistry/dm-borrower-frontend/blob/develop/tests.py